apps/jobs/views.py

```diff
@@ -389,117 +396,88 @@ def leaderboard(request, challenge_phase_split_id):
         return paginator.get_paginated_response(response_data)
 
 
-@throttle_classes([UserRateThrottle])
 @api_view(['GET'])
+@throttle_classes([UserRateThrottle])
 @permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
 @authentication_classes((ExpiringTokenAuthentication,))
 def get_remaining_submissions(request, challenge_phase_pk, challenge_pk):
-
     '''
     Returns the number of remaining submissions that a participant can
     do daily, monthly and in total to a particular challenge phase of a
     challenge.
     '''
+    response_data, response_status = get_remaining_submission_for_a_phase(request.user,
```

Comment:
I dont think we need this API now. Can we please drop this API and just use the new API that returns the remaining submission limit for all phases.

Note: You might have to check if the CLI uses these APIs. It might break if we just delete this.